RELEASE/data/autoscend_properties.txt

```diff
@@ -226,3 +226,4 @@
 223	auto_paranoia_counter
 224	auto_wishes
 225	auto_restoreUseBloodBond
+226 auto_towerBreak
```

Comment:
need a tab here instead of a space

Reply:
My bad on that one. I ran assemble.sh, I don't think it builds auto_properties.txt though (could be wrong). Fixed!